### PR TITLE
WS2-1263: Final Sass compilation for release

### PR DESCRIPTION
**Pull request by tbutterf**
Please look at this carefully. We might want to go through it together @mlsamuelson and @chill5-0. I hope I got everything in there, but, I'm not as confident as I'd like to be.

### CODE REVIEW

File: src/sass/bootstrap-asu-extends.scss [hunk 1]
```diff
@@ -1,66 +1,44 @@
 // css Bootstrap doesn't have variables for
-@import '../components/accordions/accordions';// imports cards.
-@import '../components/alerts/alerts';
-@import '../components/block/block';
-@import 'extends/alt_styling';
-@import 'extends/anchor-menu';
-@import '../components/back-to-top/back-to-top';
-@import '../components/backgrounds/backgrounds';
+@import 'extends/misc';
```
Comment (mlsamuelson) [inline src/sass/bootstrap-asu-extends.scss:2]: Looking at the bootstrap-asu-extends.scss, it appears we need to ensure updates to this file get made, but keep some custom additions, such as the imports of things in the `../components/` folder.

File: src/sass/bootstrap-asu.scss [hunk 2]
```diff
@@ -5,46 +5,49 @@
  * ======================================================================== */
 
 // Load UDS Design Tokens
-@import "design-tokens/extra-vars";
 @import 'design-tokens/variables';
 
 // default path for image assets (assuming the images will be hosted from the web root)
 // Storybook and themes override this for their particular path needs
-$image-assets-path: '../../img' !default;
+$image-assets-path: '/img' !default;
```
Comment (mlsamuelson) [inline src/sass/bootstrap-asu.scss:12]: Same as with the other "manifest" file, we need to make sure updates get made here, but also to keep the details tweaked for the CMS.

The README.md for the theme will need to be updated about these details.

File: src/sass/bootstrap-asu.scss [hunk 3]
```diff
@@ -125,9 +128,6 @@ $theme-colors: map-remove($theme-colors, 1, 3, 5);
 @import '../../node_modules/bootstrap/scss/utilities';
 @import '../../node_modules/bootstrap/scss/print';
 
-// css that is available after Bootstrap upgrade.
-@import 'bootstrap-asu-upgrade';
-
```
Comment (mlsamuelson) [inline src/sass/bootstrap-asu.scss:130]: We don't want to lose this import which handles our gap-fitting between bootstrap versions.

File: src/sass/bootstrap-asu-extends.scss [hunk 4]
```diff
@@ -1,66 +1,44 @@
...
-@import '../components/video/video';
 @import 'extends/person-profile';
-
-// Misc
-@import '../components/ckeditor/ckeditor';
-@import '../components/testimonial-carousel/testimonial-carousel';
-@import '../components/story-hero/story-hero'; // override for image.
-@import '../components/anchors/anchors';
-@import '../components/cards/card-and-image-parallax';
-@import '../components/image-text-block/image-text-block';
```
Comment (mlsamuelson) [inline src/sass/bootstrap-asu-extends.scss:66]: Imports like `@import '../components/image-text-block/image-text-block';` are going to be overrides for the CMS which we need to keep.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

File: src/components/tabbed-panels/tabbed-panels.js [hunk 5]
```diff
@@ -1,5 +1,24 @@
-(function ($) {
+import { googleAnalytics } from '../../../src/js/googleAnalytics';
```
Comment (tbutterf) [inline src/components/tabbed-panels/tabbed-panels.js:1]: @mlsamuelson This file reference doesn't work. I'm not sure if it is referring to the Google Analytics that is housed in the asu-brand module somewhere, or if this is different. Should I copy the file that is referenced in unity and place it in this theme?

Comment (tbutterf) [reply src/components/tabbed-panels/tabbed-panels.js:1]: Oh. It looks like maybe the file was renamed in WS2 to be "datalayer.js," but the two files are different. Not sure what to do there.

File: src/sass/bootstrap-asu-extends.scss [hunk 6]
```diff
@@ -1,66 +1,68 @@
 // css Bootstrap doesn't have variables for
```
Comment (tbutterf) [inline src/sass/bootstrap-asu-extends.scss:1]: I have rearranged this file to match more closely the original file in unity. It should be good to go now.

File: src/components/tabbed-panels/tabbed-panels.js (hunk 5 above)
Comment (mlsamuelson) [reply src/components/tabbed-panels/tabbed-panels.js:1]: @tbutterf That path is digging out from the CSS aggregation folder!

Comment (chill5-0) [reply src/components/tabbed-panels/tabbed-panels.js:1]: @mlsamuelson @tbutterf making a note that I created the datalayer.js file to handle the data layer attribute additions to the twig files in renovation.

**Comment by tbutterf** [comment]
I have tested the tabbed-panels.js, and it is working just fine.

**Review by mlsamuelson — APPROVED** [review]
This looks good, @tbutterf ! Thanks for working through the issues and being our guinea pig with preparing the theme for release!

**Pull request merged**